beets/autotag/mb.py

```diff
@@ -208,9 +208,12 @@ def track_info(recording, index=None, medium=None, medium_index=None,
     lyricist = []
     composer = []
     composer_sort = []
+    work = []
+    work_disambig = []
     for work_relation in recording.get('work-relation-list', ()):
         if work_relation['type'] != 'performance':
             continue
+        work_id=work_relation['work']['id']
```

Comment:
It looks like you're assigning to a new variable `work_id` here, but this is never making it into the `TrackInfo` object. You might want to use `info.work_id = ...` here.